styx-api-service/src/main/java/com/spotify/styx/api/StyxConfigResource.java

```diff
@@ -82,7 +82,7 @@ public StyxConfigResource(Storage storage) {
           Status.BAD_REQUEST.withReasonPhrase("Missing 'enabled' query parameter"));
     }
 
-    final boolean enabled = query.map(Boolean::parseBoolean).get();
+    final boolean enabled = Boolean.parseBoolean(query.get());
```

Comment:
The `query` variable name is a bit confusing. First I thought it was the entire query string and wondered why the query string itself was being parsed as a boolean.